### ISSUE TITLE
Swag update?

### ISSUE THREAD

**Issue by Aurther-Nadeem** [education/graduation#4450]

**Comment by yinkokpheng** [comment]
#4446 [Comment](https://github.com/education/graduation/issues/4446#issuecomment-651550310)

**Comment by jasperchua99** [comment]
probably should also ask whose going to be our logistic carrier too

**Comment by vimalverma558** [comment]
#4357 Comment

**Comment by yashrajjain726** [comment]
Yes, Its been 1 month and we haven't received any details regarding Swags.

**Comment by ht0rohit** [comment]
Any update now? It's been 11 more days since the last query 😁

**Comment by amansharma07** [comment]
[#4456 (comment)](https://github.com/education/graduation/issues/4456#issuecomment-663642487)
Hope this helps! :)

**Comment by AlexanderOMara** [comment]
Mine arrived today (https://github.com/education/graduation/issues/4456#issuecomment-663894825).

**Comment by Abhishekds94** [comment]
Mine arrived yesterday. I live in Illinois.
